guides/release/components/contextual-components.md

```diff
@@ -1,32 +1,32 @@
 The
 [`{{component}}`](https://www.emberjs.com/api/ember/release/classes/Ember.Templates.helpers/methods/component?anchor=component)
 helper can be used to defer the selection of a component to run time. The
-`<MyComponent />` syntax always renders the same component, while using the
+`<MyComponent/>` syntax always renders the same component, while using the
```

Comment:
@jenweber @pzuraq is this the style we should use throughout the guides? No space?

Reply:
Ya, we should have a space here. 
```suggestion
`<MyComponent />` syntax always renders the same component, while using the
```